tests/WebTest.php

```diff
@@ -154,4 +154,19 @@ public function idProvider() : iterable
         yield 'ID foo' => ['foo'];
         yield 'ID bar' => ['bar'];
     }
+
+    /**
+     * @test
+     */
+    public function it_shows_errors() : void
+    {
+        $client = self::createClient(['debug' => false]);
```

Comment:
necessity to set `debug` to `false` prevents local testing right? I see the Symfony exception page:
![Screenshot_2019-07-09 No route found for GET cdscds (404 Not Found)](https://user-images.githubusercontent.com/160299/60891390-24831100-a255-11e9-9e33-b329f03f29c6.png)

Reply:
You can test by going to `/_error/404` etc.

Reply:
Or just use `docker-compose.yaml`.

Reply:
The first works well for me.